exercises/nucleotide-count/nucleotide_count_test.go

```diff
@@ -20,6 +22,22 @@ func (h Histogram) sameMappings(o Histogram) (res bool) {
 	return
 }
 
+func assertError(count, expected int, err error, t *testing.T) {
+	var _ error = err
```

Comment:
Since you've defined a function that takes an `error` type, this line is unnecessary.

That said, I'd rather not have the function, since it's used only once, and it's hard to guess what the order of the arguments are.